tests/unittests/plugin_tests.py

```diff
@@ -388,10 +388,9 @@ def test_glob_file_over_limit(self):
         create_file(2, dir=tmpdir)
         create_file(2, dir=tmpdir)
         self.mp.add_copy_spec(tmpdir + "/*", 1)
-        self.assertEquals(len(self.mp.copy_strings), 1)
-        content, fname, _tags = self.mp.copy_strings[0]
-        self.assertTrue("tailed" in fname)
```

Comment:
We stop testing such an assert here - worth comparing filenames like 

    fn = create_file(2, dir=tmpdir)
    ..
    self.assertTrue(fname == fn)

(likewise changed in `test_single_file_over_limit`)?

Reply:
Not for this test anymore - we're testing size collection size here, and we no longer write the file to disk (so no name to compare to) within `add_copy_spec()` for files over the limit any more. We'd have to have this test go all the way thru `collect_plugin()` now to test the file naming, which is still handled elsewhere.